### PR TITLE
Add quickstart guide

### PR DESCRIPTION
**Pull request by cdrage**
Adds a quickstart guide which will appear as the index page on the
website.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- name: httpd
+  type: NodePort
+    ports:
+      - port: 8080
+          targetPort: 80
+```
+
+__2. Now run the deploy command to deploy to Kubernetes!__
+
+``sh
```
Comment (kadel) [inline docs/quickstart.md]: missing `` ` ``

File: docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Type:                   NodePort
+IP:                     10.0.0.34
+Port:                   <unset> 8080/TCP
+NodePort:               <unset> 31511/TCP
+Endpoints:              172.17.0.4:80
+Session Affinity:       None
+No events.
+```
+
+If you're using a cloud provider, your IP will be listed next to `LoadBalancer Ingress`.
```
Comment (kadel) [inline docs/quickstart.md]: Don't know if this is true. :-( 

this is output that I see when deployed to GKE

```
▶ kubectl describe svc httpd
Name:			httpd
Namespace:		default
Labels:			app=httpd
Annotations:		<none>
Selector:		app=httpd
Type:			NodePort
IP:			10.55.241.178
Port:			<unset>	8080/TCP
NodePort:		<unset>	31381/TCP
Endpoints:		10.52.1.6:80
Session Affinity:	None
Events:			<none>
```

Comment (kadel) [inline docs/quickstart.md]: It might be better to use LoadBalancer or Ingress in our examples.

Comment (cdrage) [inline docs/quickstart.md]: Woops. Looks like I need to fix this in the quickstart for kedge as well as kompose.

Comment (cdrage) [inline docs/quickstart.md]: Ahhhh I see, in our example we use NodePort but Kompose we use LoadBalancer, hence the comment.

**Comment by cdrage** [comment]
@kadel I've gone ahead and removed the line about the Cloud Provider as well as added the missing `

**Pull request merged**